client/src/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="Thoughts of the day challenge">
+  <meta name="author" content="Alex">
+
+  <title>The Wall</title>
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css" />
```

Comment:
@alexander-daniel Could we include this inside of the webpack "build" bundle? If so, how?

Reply:
Yep, you could include a css-loader and import it in your JS. You could also do the same so that this HTML file is generated using a template and webpack and the asset links are automatically injected in.